web/src/SidebarItemView.tsx

```diff
@@ -336,9 +336,17 @@ export function EnabledSidebarItemView(props: SidebarItemViewProps) {
   let onStartBuild = startBuild.bind(null, item.name)
   const groupViewIndentClass = props.groupView ? "groupViewIndent" : ""
   let analyticsTags = { target: item.targetType }
+  let ref: any = useRef(null)
```

Comment:
could this be typed as `MutableRefObject<HTMLLIElement | null>`?

Reply:
oooh nice suggestion! done